src/test/java/stepDefinitions/StepsHome.java

```diff
@@ -0,0 +1,31 @@
+package stepDefinitions;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.github.cdimascio.dotenv.Dotenv;
+import pages.HomePage;
+import pages.NavBarPage;
+import utils.BrowserManagement;
+
+public class StepsHome {
+
+    private HomePage homePage;
+    private NavBarPage navBarPage;
+    private static Dotenv dotenv = Dotenv.load();
+
+    public StepsHome() {
+        this.homePage = new HomePage();
+        this.navBarPage = new NavBarPage();
+    }
+
+    @Given("^that I enter the conduit site$")
+    public void enterConduitSite() {
+        BrowserManagement.getDriver().get(dotenv.get("BASE_URL"));
```

Comment:
Move this line to a page and call that method from here

Reply:
It is moved to the BasePage class and the method is called from the step
`public void enterTheWebsite(){    driver.get(dotenv.get("BASE_URL"));     }`